### DISCUSSION TITLE
Get the Current User in Data Source create and read

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#9840] [Help] [answered]
Hello, I want to get the details of the `Current User` who is accessing a custom datasource plugin .... specifically in the `create` and `read` functions. What is the best way to get these details on the server side (i.e. without any manipulation possible from the browser console)

**Comment by melohagan** [comment, marked as answer]
Hey @SantoshSrinivas79 

I believe I answered your question on Discord.

I also found this post on Discord that may also provide some additional information: https://ptb.discord.com/channels/733030666647765003/1042791429115748352
